tests/test_serde_objects.rs

```diff
@@ -0,0 +1,65 @@
+use serde::{Deserialize, Serialize};
+use std::thread;
+use std::thread::JoinHandle;
+use std::time;
+
+mod helpers;
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq)]
+struct LargeObject {
+    field: String,
+    fields: Vec<String>,
+}
+
+#[test]
+fn test_serde_objects() {
+    let mcrouter = helpers::connect("memcache://localhost:11311?protocol=ascii").unwrap();
+
+    let key = "large_obj";
+
+    for i in 0..20 {
+        let object = LargeObject {
+            field: format!("{}", i + 29).repeat(2000),
+            fields: vec![format!("x{}>{}", i, i); 2002],
```

Comment:
Is this array actually large? I understand it creates a vector with a capacity of 2002, doesn't it contain just a single value?

Reply:
it contains 2002 values of first argument